spec/01-unit/31-simdjson/01-cjson_compatibility_spec.lua

```diff
@@ -0,0 +1,284 @@
+local helpers = require("spec.helpers")
+local cjson = require("cjson.safe")
+local simdjson = require("resty.simdjson")
+
+
+local deep_sort = helpers.deep_sort
+
+
+describe("[cjson compatibility] examples from cjson repo", function ()
+
+  local strs = {
+[[
+{
+    "glossary": {
+        "title": "example glossary",
+                "GlossDiv": {
+            "title": "S",
+                        "GlossList": {
+                "GlossEntry": {
+                    "ID": "SGML",
+                                        "SortAs": "SGML",
+                                        "GlossTerm": "Standard Generalized Mark up Language",
+                                        "Acronym": "SGML",
+                                        "Abbrev": "ISO 8879:1986",
+                                        "GlossDef": {
+                        "para": "A meta-markup language, used to create markup languages such as DocBook.",
+                                                "GlossSeeAlso": ["GML", "XML"]
+                    },
+                                        "GlossSee": "markup"
+                }
+            }
+        }
+    }
+}
+]],
+
+[[
+{"menu": {
+  "id": "file",
+  "value": "File",
+  "popup": {
+    "menuitem": [
+      {"value": "New", "onclick": "CreateNewDoc()"},
+      {"value": "Open", "onclick": "OpenDoc()"},
+      {"value": "Close", "onclick": "CloseDoc()"}
+    ]
+  }
+}}
+]],
+
+[[
+{"widget": {
+    "debug": "on",
```

Comment:
Some indentations use two spaces, while others use four spaces

Reply:
No, we will keep the original data format.